index.html

```diff
@@ -4197,16 +4197,20 @@ <h3>Securing Mechanism Specifications</h3>
         </p>
 
         <p>
-Securing mechanism specifications MUST provide a verification mechanism that
+Securing mechanism specifications MUST provide a verification algorithm that
 returns only the information in the [=conforming document=] that has been
 secured, without any securing mechanism information included, such as `proof` or
```

Comment:
I'm not sure if anyone else would get tripped up by this, but in reading this description it sounded to me like it would be a contradiction to return additional information below about the securing mechanism -- since this says "returns only... without any securing mechanism information included...". So a minor reshuffling here to indicate the secured information must be returned "in isolation" might help:

```suggestion
Securing mechanism specifications MUST provide a verification algorithm that
returns the information in the [=conforming document=] that has been
secured in isolation, without any securing mechanism information included, such as `proof` or
```